package.json

```diff
@@ -32,7 +32,8 @@
         "hapi": ">=17"
     },
     "dependencies": {
-        "accept": "^3.0.2"
+        "accept": "^3.0.2",
+        "hapi-auth-cookie": "^9.0.0"
```

Comment:
This is a dev dependency used only in the tests.

Reply:
Fixed